project.owasp.yaml

```
@@ -0,0 +1,279 @@
+audience:
+  - builder
+  - defender
+community:
+  - name: proj-dependency-check
+    platform: slack
+    url: https://owasp.slack.com/archives/C2FTJJ26N
+    description: channel for the https://www.owasp.org/index.php/OWASP_Dependency_Check project - For general questions with the tool, check out the mailing
+      list https://groups.google.com/forum/#!forum/dependency-check. To report issues, please submit a GitHub issue.
```

```suggestion
      list https://groups.google.com/g/dependency-check. To report issues, please submit a GitHub issue.
```

This does seem a bit dead though. Maybe we should prefer https://github.com/dependency-check/DependencyCheck/discussions now.